src/software/gui/robot_diagnostics/widgets/feedback.h

```diff
@@ -5,10 +5,19 @@
 // This include is autogenerated by the .ui file in the same folder
 // The generated version will be names 'ui_<filename>.h'
 #include "software/gui/robot_diagnostics/ui/ui_main_widget.h"
+#include "software/proto/sensor_msg.pb.h"
 
 /**
  * Set up feedback area of the widget
  *
  * @param widget widget to setup
  */
 void setupFeedback(Ui::AutoGeneratedMainWidget *widget);
+
+/**
+ * Update values of components in the feedback widget using a new SensorMsg
```

Comment:
I got a bit confused at first with the term "feedback" because I didn't realize we'd named the widget with all the bars/values the `feedback_widget`. I initially thought feedback might refer to something else. Possibly out of scope for this PR, but I think it would be good if we could think of another name for this widget that's a bit more clear.

Reply:
maybe ``` sensor_widget ``` or ``` sensor_status_widget ```?

Reply:
I like `sensor_status_widget` thats pretty clear

Reply:
I like `sensor_status_widget` too. I ported over `feedback` from mrftest, so that's why it was called that